server/jetstream_api.go

```diff
@@ -2341,6 +2361,92 @@ func (s *Server) jsLeaderServerStreamMoveRequest(sub *subscription, c *client, _
 
 	cfg.Placement = origPlacement
 
+	s.Noticef("Requested move of: R=%d stream %s in account %s from old peer set %+v to new peer set %+v",
```

Comment:
Derek often recommends the use of `'%s > %s'` (with account and stream) to be consistent when we issue JS log statements that contain those.

Reply:
fixed